### PR TITLE
Find missing ingredients + test

### PR DESCRIPTION
**Pull request by jsleslie**

### CODE REVIEW

File: R/find_missing_ingredients.R [hunk 1]
```diff
@@ -1,17 +1,26 @@
 #' find_missing_ingredients Documentation
 #'
-#' For each column with missing values, this function will create a reference
-#' list of row indices, sum the number and calculate proportion of missing values
+#' Example Description: Returns the indices of rows with columns which have missing values,
+#' as well as how many missing values they have
 #'
-#' @param df A dataframe
+#' @param data
 #'
```
Comment (mglu123) [inline R/find_missing_ingredients.R]:
```suggestion
#' @param data A data frame that want to check missing values
#'
```

Add explanation to the parameter.

File: R/find_missing_ingredients.R [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
-#' @return A dataframe containing the columns: column_name, list_of_indices, na_count, na_proportion
+#' @return
```
Comment (mglu123) [inline R/find_missing_ingredients.R]: What will be returned?

**Review by mglu123 — DISMISSED** [review]
Some minor changes need in documentations. Others are good!

File: R/find_missing_ingredients.R (hunk 2 above)
Comment (jsleslie) [reply R/find_missing_ingredients.R]: Addressed now!

**Review by jsleslie — COMMENTED** [review]
Feedback addressed

**Review by mglu123 — APPROVED** [review]

**Pull request merged**